etcdctl/ctlv2/command/util.go

```diff
@@ -78,7 +79,7 @@ func getPeersFlagValue(c *cli.Context) []string {
 
 	// If we still don't have peers, use a default
 	if peerstr == "" {
-		peerstr = "http://127.0.0.1:2379,http://127.0.0.1:4001"
```

Comment:
Also what i feel is most people going for heavy usage of etcd will mostly be having linux system where in etcdserver startup it displays to logs in terms of 127.0.0.1:2379, hence changing etcdctlv2 to localhost for all may make it non-uniform with the logs and might even confuse users.